### DISCUSSION TITLE
Welcome to Oritech Discussions!

### DISCUSSION THREAD

**Discussion by Rearth** [Rearth/Oritech#2] [Announcements]
## 👋 Welcome!
  Oritech uses discussions to plan future content and organize open topics that different people might be working on.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Getting Started:
If you want to contribute to oritech, feel free to get started on the open topics listed in the corresponding discussion channels

**Comment by MinejekHD** [comment]
Hello, is there a chance that this mod will be available for Forge 1.20.1 in the future?

**Reply by Rearth** [reply]
Hi, no 1.20.1 support is currently not planned.

**Comment by Shizzo27** [comment]
Hoi, is there any plans on updating to 1.21.8 ?
